certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/common/codesystems/CodeSystemKvIntygstyp.java

```diff
@@ -0,0 +1,86 @@
+package se.inera.intyg.certificateservice.infrastructure.certificatemodel.common.codesystems;
+
+import se.inera.intyg.certificateservice.domain.common.model.Code;
+
+public class CodeSystemKvIntygstyp {
```

Comment:
I like the way you added the code system for certificate types!

There are minor discrepancies with the published code system and also I noticed that Inera haven't published the new codes for the new certificates.

Would be great if we verified with requirements to be sure that we are naming the codes correctly. Here is the link to what is currently published (use the oid to identify the correct one, as there are two related to intyg): https://inera.atlassian.net/wiki/spaces/KINT/pages/2648506471/Kodverk+och+urval+i+de+nationella+tj+nstekontrakten